Tic-Tac-Toe/Cell.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace TicTacToe
+{
+    internal class Cell
+    {
+        public int Row { get; set; }
+
+        public int Column { get; set; }
+
+        private Cell(int row, int column)
+        {
+            Row = row;
+            Column = column;
+        }
+
+        public static Cell From(int row, int column)
+        {
+            return new Cell(row, column);
+        }
+
+        public static Cell ErrorCell()
+        {
```

Comment:
Use a regular public constructor.
This can be used when creating a complex object